randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1017,8 +1035,11 @@ Extra - room_id: [25]
           All of the following:
               Gravity Suit
               Any of the following:
-                  Space Jump or Wall Jump (Intermediate)
+                  Space Jump
+                  # WJ with HJ: https://youtu.be/vku11Eov-m0
                   Hi-Jump and Wall Jump (Beginner)
+                  # WJ without HJ: https://youtu.be/TDwTWYNa_F8
+                  Wall Jump (Intermediate)
```

Comment:
looking at these video side by side, the WJs look basically the exact same to me. is this really worth a bump? Feels like it should always just be WJ beginner.

Reply:
WJ beginner should be ~1 WJ, which you can do with HJ but not without. not a big deal if we consolidate to me

Reply:
the total amount of walljumps are basically:tm: the same. i'd say we just consolidate to beginner.